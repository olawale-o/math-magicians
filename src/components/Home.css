.Home {
  padding: 15px;
}

h4 {
  font-size: 22px;
}

p {
  margin-top: 20px;
  font-size: 18px;
}
