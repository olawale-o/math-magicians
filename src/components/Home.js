import React from 'react';
import './Home.css';

const Home = () => (
  <div className="Home">
    <h4>Welcome to our page!</h4>
    <p>
      It is a long established fact that a reader will be distracted by the readable
      content of a page when looking at its layout.
      The point of using Lorem Ipsum is that it has a more-or-less normal distribution of
      letters, as opposed to using &nbsp;
      <q>
        Content here, content here
      </q>
      &nbsp;
      making it look
      like readable English.
      Many desktop publishing packages and web page editors now use Lorem Ipsum
      as their default model text,
      and a search for
      <q>lorem ipsum</q>
      &nbsp;
      will uncover many web sites still in their infancy.
      Various versions have evolved over the years,
      sometimes by accident, sometimes on purpose (injected humour and the like).
    </p>
    <p>
      It is a long established fact that a reader will be distracted by the readable
      content of a page when looking at its layout.
      The point of using Lorem Ipsum is that it has a more-or-less normal distribution of
      letters, as opposed to using &nbsp;
      <q>
        Content here, content here
      </q>
      &nbsp;
      making it look like readable English.
      Many desktop publishing packages and web page editors now use Lorem Ipsum
      as their default model text, and a search for &nbsp;
      <q>lorem ipsum</q>
      &nbsp;
      will uncover many web sites still in their infancy.
      Various versions have evolved over the years,
      sometimes by accident, sometimes on purpose (injected humour and the like).
    </p>
  </div>
);

export default Home;
