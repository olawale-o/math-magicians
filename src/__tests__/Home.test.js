import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Home', () => {
  it('Renders Home component correctly and create snapshot', () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('it has a header Element', () => {
    render(<Home />);

    const headerEle = screen.getByRole('heading', {
      name: 'Welcome to our page!',
    });

    expect(headerEle).toBeTruthy();
  });
});
