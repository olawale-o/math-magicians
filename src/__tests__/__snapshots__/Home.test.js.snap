// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Renders Calculator component correctly and create snapshot 1`] = `
<div
  className="Home"
>
  <h4>
    Welcome to our page!
  </h4>
  <p>
    It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using  
    <q>
      Content here, content here
    </q>
      making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for
    <q>
      lorem ipsum
    </q>
      will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
  </p>
  <p>
    It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using  
    <q>
      Content here, content here
    </q>
      making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for  
    <q>
      lorem ipsum
    </q>
      will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
  </p>
</div>
`;

exports[`Home Renders Home component correctly and create snapshot 1`] = `
<div
  className="Home"
>
  <h4>
    Welcome to our page!
  </h4>
  <p>
    It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using  
    <q>
      Content here, content here
    </q>
      making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for
    <q>
      lorem ipsum
    </q>
      will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
  </p>
  <p>
    It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using  
    <q>
      Content here, content here
    </q>
      making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for  
    <q>
      lorem ipsum
    </q>
      will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
  </p>
</div>
`;
